apps/frontend/src/pages/imports/devcon.tsx

```diff
@@ -0,0 +1,151 @@
+import { AppButton } from "@/components/ui/Button";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import { toast } from "sonner";
+import { SupportToast } from "@/components/ui/SupportToast";
+import { errorToString } from "@types";
+import { ERROR_SUPPORT_CONTACT } from "@/constants";
+import { devFetchSchedule } from "@/lib/imports/integrations/devcon";
+import { updateUserData } from "@/lib/storage/localStorage/user/userData";
+import { CursiveLogo } from "@/components/ui/HeaderCover";
+import Link from "next/link";
+
+
+type UsernameSubmission = {
+  username: string;
+};
+
+async function FetchAndSaveDevconSchedule(username: string): Promise<boolean> {
+  const { user } = await storage.getUserAndSession();
+
+  // Fetch schedule
+  const schedule = await devFetchSchedule(username)
+  if (!schedule) {
+    return false;
+  }
+
+  // Update userdata
+  const userData: UserData = user.userData;
+
+  if (!userData.devcon) {
+    userData.devcon = {};
+  }
+  userData.devcon.username = username;
+  userData.devcon.schedule = schedule;
+
+  await updateUserData(userData);
+  return true;
+}
+
+const DevconScheduleImportPage = () => {
+  const router = useRouter();
+  const [loading, setLoading] = useState(false);
+  const { register, handleSubmit } =  useForm<UsernameSubmission>();
+
+
+  const onHandleSubmit = async (formData: UsernameSubmission) => {
+    setLoading(true);
+    const {
+      username,
+    } = formData;
+    try {
+      const succeeded = await FetchAndSaveDevconSchedule(username);
+      if (!succeeded) {
+        toast.error("No schedule imported, are you sure your username was correct?")
+        setLoading(false);
+        return;
+      }
+      toast.success("Successfully imported Devcon schedule.");
+      router.push("/profile");
+    } catch (error) {
+      console.error(error);
+      toast(
+        SupportToast(
+          "",
+          true,
+          "Error importing Devcon schedule.",
+          ERROR_SUPPORT_CONTACT,
+          errorToString(error)
+        )
+      );
+    }
+    setLoading(false);
+  };
+
+  return (
+    <>
+      { !loading ? (
+      <AppLayout
+        back={{
+          href: "/profile",
+          label: "Back",
+        }}
+        headerDivider
+        showFooter={false}
+      >
+        <form
+          onSubmit={handleSubmit(onHandleSubmit)}
+          className="flex flex-col gap-4 pt-4 pb-6 px-2"
+        >
+          <div className="flex items-center justify-between gap-4">
+            <div className="flex flex-col gap-2">
+              <span className="text-label-primary">
+                <b>Submit</b> your Devcon username to import your schedule.
+              </span>
+              <span className="text-[16px] text-label-secondary py-3">
+                To import your schedule there are a few manual steps:
+                <br/>
+                <ul>
+                  <li>
+                    - Navigate to your account settings.
```

Comment:
remove the dash here, the li should just create bullets normally?

Reply:
when I removed the dashes, bullets were not automatically created.